### ISSUE TITLE
Problem with tsconfig.json when using new 1.8 options

### ISSUE THREAD

**Issue by wialy** [Microsoft/TypeScript-Sublime-Plugin#447]
Currently, **when I try to set some new options like `allowUnreachableCode` in my `tsconfig.json` file** the Sublime Typescript Plugin (_which still uses 1.7.3 compiler, why not the one installed with the project or global?_) ignores it totally, like there's no `tsconfig` at all.

As a result, there are errors like `Cannot compile modules unless '--module' flag is provided` in Sublime even with such `package.json` file:

```
{
	"compilerOptions": {
		"module": "commonjs",
		"jsx": "React",
		"target": "ES6",
		"sourceMap": false,
		"allowUnreachableCode": true
	},
	"exclude": [
		"node_modules",
		"built",
		"typings/main",
		"typings/main.d.ts"
	]
}
```

The projects build completes with no errors when running `tsc` direct from console.

Also I have noticed that any misspelled option causes the same behaviour and settings other options from `tsconfig.json` fails.

**Comment by billti** [comment]
Yeah, it is a pain when "*errors*" in a `tsconfig.json` file cause it to not get applied. https://github.com/Microsoft/TypeScript/issues/6600 is tracking improving this.

The Sublime plugin will be re-released shortly to include the latest 1.8 TypeScript compiler (just got merged in 30 mins ago, should be pushed out some time today).

**Comment by wialy** [comment]
@billti Thanks for you reply.

An error message or warning from plugin when anything's wrong with `tsconfig` could help to avoid such situations. Also, hope to see a `tsconfig` watcher some day.

Thanks for your work, love TypeScript and Sublime and happy to see them together :)
